pkg/distributor/distributor.go

```diff
@@ -77,6 +77,9 @@ const (
 	// Size of "slab" when using pooled buffers for marshaling write requests. When handling single Push request
 	// buffers for multiple write requests sent to ingesters will be allocated from single "slab", if there is enough space.
 	writeRequestSlabPoolSize = 512 * 1024
+
+	// until http.StatusServiceOverload exists
+	statusServiceOverload = 529
```

Comment:
after writing previous suggestion, I've realized that this is not "official" HTTP status code, so `http.StatusServiceOverload` may never exist. :( we could fix the comment:

```suggestion
	// 529 is non-standard status code used by some services to signal that "The service is overloaded".
	statusServiceOverload = 529
```